### PR TITLE
Cleanup documentation on deploying from your laptop

### PR DESCRIPTION
**Pull request by yuvipanda**
Ref https://github.com/2i2c-org/pilot-hubs/issues/351

### CODE REVIEW

**Review by damianavila — APPROVED** [review]
LGTM, this is great @yuvipanda! Thanks for adding those details, it is pretty clear now!

**Comment by damianavila** [comment]
Just a general related question/comment (maybe silly and minimal... and this is not a blocker for merging this PR).
But having the script called `deploy` with the same name as the most important subcommand `deploy` seems prone to confuse people, IMHO (ie. Is this a copypaste error in the command?)

   ```bash
   python3 deploy deploy <cluster-name> <hub-name>
   ```

Have you discussed before about, maybe, renaming the script to something such as `deployer` (or whatever name you want except for `deploy` 😉 ).

**Comment by yuvipanda** [comment]
@damianavila to avoid confusion, I've now renamed the `deploy` directory, so to deploy you will now have to do:

```bash
python3 python3 deploy 2i2c staging
```

I hope this is less confusing?

**Comment by choldgraf** [comment]
is the 2 `python3`s intentional?

**Comment by damianavila** [comment]
> is the 2 python3s intentional?

Je... maybe @yuvipanda copy-pasted the wrong word or I am bad at understanding jokes 😜

**Comment by yuvipanda** [comment]
@damianavila @choldgraf it was, indeed, a joke :D Instead of deploy being repeated twice because it's the name of the dir and subcommand, I got python3 to repeat twice by renaming the dir to python3 :) 

Now I've done something *useful* instead and taken @damianavila's naming suggestion - the dir is now called `deployer`.

**Comment by choldgraf** [comment]
lol @yuvipanda that was a solid troll

so the new command would be

```
python3 deployer deploy 2i2c staging
```

**Comment by yuvipanda** [comment]
@choldgraf yes that's correct! I updated the docs too

File: docs/howto/operate/manual-deploy.md [hunk 1]
```diff
@@ -1,52 +1,74 @@
...
+need to deploy from your laptop - primarily when testing changes on staging or
+actively fixing an outage. This isn't ideal, but this is where we are now.
 
-## Deploy script arguments and usage
-Running:
-```
-python3 deploy.py deploy --help
-```
-Will provide information about the `deploy.py` deploy options:
+Our deployment scripts live in the [`deploy/`](https://github.com/2i2c-org/pilot-hubs/blob/master/deploy.py)
```
Comment (choldgraf) [inline docs/howto/operate/manual-deploy.md:7]:
```suggestion
Our deployment scripts live in the [`deploy/` folder](https://github.com/2i2c-org/pilot-hubs/blob/master/deploy.py)
```

File: docs/howto/operate/manual-deploy.md [hunk 2]
```diff
@@ -1,52 +1,74 @@
...
-    ```
-* `hub_name` argument
+2. Install python packages required by our deployment script
+   
+   ```bash
+   pip install -r requirements.txt
+   ```
 
-    Deploys only the hub passed as argument. 
+3. Install the [Google Cloud SDK](https://cloud.google.com/sdk) so
```
Comment (choldgraf) [inline docs/howto/operate/manual-deploy.md:28]: What about for hubs that are on AWS or Azure? If those aren't supported right now, we should add a `note:` block to make it clear

**Review by choldgraf — COMMENTED** [review]
Just two quick comments, but this looks good to me!

File: docs/howto/operate/manual-deploy.md (hunk 1 above)
Comment (damianavila) [reply docs/howto/operate/manual-deploy.md:7]: That URL link should be https://github.com/2i2c-org/pilot-hubs/blob/master/deployer with the latest change.

File: docs/howto/operate/manual-deploy.md [hunk 3]
```diff
@@ -1,52 +1,74 @@
...
-to be usable with it before you can do deployments.
\ No newline at end of file
+## Doing a deploy
+
+1. Make the [config change](../topic/config.md) you want to deploy.
+
+2. Deploy just a single hub:
+
+   ```bash
+   python3 deployer deploy <cluster-name> <hub-name>
```
Comment (damianavila) [inline docs/howto/operate/manual-deploy.md:60]: ❤️

File: docs/howto/operate/manual-deploy.md (hunk 2 above)
Comment (yuvipanda) [reply docs/howto/operate/manual-deploy.md:28]: I've updated the text here to clarify.

File: docs/howto/operate/manual-deploy.md (hunk 1 above)
Comment (yuvipanda) [reply docs/howto/operate/manual-deploy.md]: Done!

**Comment by yuvipanda** [comment]
@damianavila @choldgraf I think I've addressed your comments. I've also removed other references to deploy.py I had missed earlier, and merged contents of test-hub.md into manual-deploy.md

**Comment by damianavila** [comment]
Super, thanks for the latest commits, LGTM again.

File: docs/howto/configure/update-env.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ have to manually build & push it after making a change.
 
 2. Make the changes you need to make to the environment, and git commit it.
 
-3. Run `python3 deploy.py build`. This will build the image and push it to
+3. Run `python3 deployer build`. This will build the image and push it to
```
Comment (GeorgianaElena) [inline docs/howto/configure/update-env.md:16]: I think the build command needs to be passed the cluster-name too.

**Review by GeorgianaElena — APPROVED** [review]
LGTM :rocket:

P.S. I love the new **deployer** name :D

File: docs/howto/configure/update-env.md (hunk 4 above)
Comment (damianavila) [reply docs/howto/configure/update-env.md:16]: That is the recent merge #358, right?

Comment (GeorgianaElena) [reply docs/howto/configure/update-env.md:16]: yes

Comment (yuvipanda) [reply docs/howto/configure/update-env.md]: Good catch, fixed.

**Comment by yuvipanda** [comment]
ok, I've rebased to resolve the merge conflicts now :)

**Comment by choldgraf** [comment]
wooo thanks for the discussion all!

**Pull request merged**